src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -613,26 +613,36 @@ PoissonLogLikelihoodWithLinearModelForMeanAndProjData<TargetT>::
 compute_sub_gradient_without_penalty_plus_sensitivity(TargetT& gradient, 
                                                       const TargetT &current_estimate, 
                                                       const int subset_num)
+{
+  actual_compute_sub_gradient_without_penalty(gradient, current_estimate, subset_num, false);
+}
+
+template<typename TargetT>
+void
+PoissonLogLikelihoodWithLinearModelForMeanAndProjData<TargetT>::
+actual_compute_sub_gradient_without_penalty(TargetT& gradient,
+                                            const TargetT &current_estimate,
+                                            const int subset_num,
+                                            const bool do_subtraction)
 {
   assert(subset_num>=0);
   assert(subset_num<this->num_subsets);
-  distributable_compute_gradient(this->projector_pair_ptr->get_forward_projector_sptr(), 
-                                 this->projector_pair_ptr->get_back_projector_sptr(), 
+  distributable_compute_gradient(this->projector_pair_ptr->get_forward_projector_sptr(),
+                                 this->projector_pair_ptr->get_back_projector_sptr(),
```

Comment:
bunch of white-space diffs? Let's get rid of them

Reply:
I believe resolved but need to check back before merge